tests/foreman/maintain/test_service.py

```diff
@@ -0,0 +1,267 @@
+"""Test class for satellite-maintain service functionality
+
+:Requirement: foreman-maintain
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: ForemanMaintain
+
+:Assignee: gtalreja
+
+:TestType: Functional
+
+:CaseImportance: Critical
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_string
+
+from robottelo.config import settings
+from robottelo.constants import HAMMER_CONFIG
+from robottelo.constants import maintain_hammer_yml
+from robottelo.constants import satellite_answer_file
+from robottelo.utils.issue_handlers import is_open
+
+pytestmark = pytest.mark.destructive
+
+
+@pytest.fixture
+def missing_hammer_config(request, sat_maintain):
+    """Setup/teardown fixture used by test_positive_service_restart_without_hammer_config"""
+    flag = 'initial_admin_password'
+    sat_maintain.execute(
+        f'sed -i "s/{flag}: {settings.server.admin_password}/{flag}: invalid/g" '
+        f'{satellite_answer_file}'
+    )
+    sat_maintain.execute(f'mv {HAMMER_CONFIG} /tmp/foreman.yml')
+    sat_maintain.execute(f'mv {maintain_hammer_yml} /tmp/foreman-maintain-hammer.yml')
+
+    @request.addfinalizer
+    def _finalize():
+        sat_maintain.execute(
+            f'sed -i "s/{flag}: invalid/{flag}: {settings.server.admin_password}/g" '
```

Comment:
```suggestion
            f'sed -i "s/{flag}: .*/{flag}: {settings.server.admin_password}/g" '
```
Match any character than fixed string to return to the original value.

Reply:
Nice suggestion, Thank you @jyejare